### DISCUSSION TITLE
AngleSharp limited to only 0.17.1

### DISCUSSION THREAD

**Discussion by ghost** [mganss/HtmlSanitizer#559] [Q&A] [answered]
I need to be able to target 4.8 framework but to do so AngleSharp is limited to only 0.17.1.  Can we get this addressed?

**Comment by mganss** [comment]
This is not specific to net48. In order to use AngleSharp 1.0, you'll have to use the prerelease version of HtmlSanitizer. See #491 for an in-depth discussion.

**Reply by ghost** [reply]
Where is the prerelease located?  I have some vision issues and might have overlooked.

**Reply by mganss** [reply]
Current prerelease is at https://www.nuget.org/packages/HtmlSanitizer/8.2.871-beta.

In VS, check the "Include prerelease versions" box near the top in the NuGet packages window.

**Comment by ghost** [comment]
Thank you

**Comment by ghost** [comment]
Any idea on when it might be moving out of beta into release

**Reply by mganss** [reply, marked as answer]
Unfortunately, no. As soon as AngleSharp.Css releases a stable version, I'll release a stable version of HtmlSanitizer, too. There's no code in HtmlSanitizer itself that would keep us from releasing a stable version.

**Reply by ghost** [reply]
Thank you for the information Michael. I really like this tool.
